### ISSUE TITLE
System uptime error on first dashboard

### ISSUE THREAD

**Issue by Alvarogtc** [xtreamui/XtreamUI#19]
The first dashboard is showing system uptime equal  SUM of all servers.

**Comment by xtreamui** [comment]
Yeah, total system uptime. Each individual dashboard shows the uptime of that individual server. What would you rather? Average uptime?

**Comment by elitetvx** [comment]
After login dashboard showing  on uptime the sum of all servers
Is that correct? My showing 3500 days
